src/applications/vaos/appointment-list/components/AppointmentsPageV2/index.jsx

```diff
@@ -75,6 +75,28 @@ export default function AppointmentsPageV2() {
     [subPageTitle],
   );
 
+  const [documentTitle, setDocumentTitle] = useState();
+  useEffect(
+    () => {
+      function handleBeforePrint(_event) {
+        document.title = `Your appointments | ${pageTitle} | Veterans Affairs`;
+      }
+
+      function handleAfterPrint(_event) {
+        document.title = documentTitle;
+      }
+      setDocumentTitle(document.title);
+
+      window.addEventListener('beforeprint', handleBeforePrint);
+      window.addEventListener('afterprint', handleAfterPrint);
+      return () => {
+        window.removeEventListener('beforeprint', handleBeforePrint);
+        window.removeEventListener('afterprint', handleAfterPrint);
+      };
+    },
+    [documentTitle, subPageTitle],
+  );
+
```

Comment:
Great solution to always print "Your appointments | VA Online Scheduling | Veterans Affairs" as the header in the print version of upcoming/requested/past/cancel appointment list.